server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -63,6 +66,8 @@
 import java.util.function.Supplier;
 import java.util.stream.Collectors;
 
+import reactor.util.annotation.NonNull;
```

Comment:
This looks like new-library dependency.
Can we follow existing conventions like `Objects.requireNonNull` ?

Reply:
Ack, will fix